### DISCUSSION TITLE
Issue with QHierarchy extension

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9090] [General - Components / Directives / etc]
I have installed the QHierarchy extension and I m trying to use it nut it is not working 
I m not getting table just getting empty space only. 
@pratik227 can u pls help 
I saw u have developed this extension.

**Comment by pratik227** [comment]
What version of quasar you are using?

**Reply by bagladivyang03** [reply]
Latest one ig its 2.2.4

**Reply by bagladivyang03** [reply]
while making the project i made quasar CLI directly without any version specifically so ig its latest one

**Reply by pratik227** [reply]
Hi @bagladivyang03,  Ii's not migrated to vue 3 Yet.

**Reply by bagladivyang03** [reply]
So I will not be able to use this ?

**Reply by pratik227** [reply]
Currently no. I will migrate it soon.

**Reply by bagladivyang03** [reply]
Any other alternatives is there to use this ?

**Reply by bagladivyang03** [reply]
> Currently no. I will migrate it soon.

Okay lmk if u are done I m currently working on some project and it's urgent to do this.

**Reply by pratik227** [reply]
I think there are no alternatives for now. Because quasar 2.2.4 is built with Vue 3 and we need to migrate all the extensions to vue3 3.

**Reply by pratik227** [reply]
> > Currently no. I will migrate it soon.
> 
> Okay lmk if u are done I m currently working on some project and it's urgent to do this.

Sure

**Reply by bagladivyang03** [reply]
Is it possible to shift my current project to low version or something ?

**Reply by bagladivyang03** [reply]
@pratik227 randomly I tried again and it works for me thanks :)
I have one more doubt like in q-table we have slots to customize the row data is there any option qhierarchy too to do  this ?

**Reply by pratik227** [reply]
Ok, yes it's there in documentation

**Reply by bagladivyang03** [reply]
Yeah thank u so much for the help
